applications/filestore-backup/values.yaml

```diff
@@ -0,0 +1,67 @@
+# Default values for filestore-backup.
+
+# -- Override the base name for resources
+nameOverride: ""
+
+# -- Override the full name for resources (includes the release name)
+fullnameOverride: ""
+
+# -- Resource limits and requests for the filestore-backup pods
+resources: {}
+
+# -- Annotations for the filestore-backup pods
+podAnnotations: {}
+
+# -- Node selector rules for the filestore-backup pods
+nodeSelector: {}
+
+# -- Tolerations for the filestore-backup pods
+tolerations: []
+
+# -- Affinity rules for the filestore-backup pods
+affinity: {}
+
+# The following will be set by parameters injected by Argo CD and should not
+# be set in the individual environment values files.
+image:
+  # -- Filestore-Backup image to use
+  repository: ghcr.io/lsst-sqre/rubin-google-filestore-tool
+  # -- Pull policy for the filestore-backup image
+  pullPolicy: "IfNotPresent"
+  # -- Tag of filestore-backup image to use
+  # @default -- The appVersion of the chart
+  tag: ""
+
+tool:
+  # -- Filestore instance (e.g. "fshare-instance-dev")
+  # @default -- Must be overridden in environment-specific values file
+  instance: ""
+  # -- Zone for Filestore instance (e.g. "b" from "us-central1-b")
+  # @default -- Must be overridden in environment-specific values file
+  zone: ""
+  # -- File Share name for filestore instance.  Always "share1" unless
+  # storage is on an Enterprise tier
+  fileShare: "share1"
+  backup:
+    # -- Turn on debugging mode
+    debug: false
+    # -- Backup schedule
```

Comment:
For the schedules, it would be nice to add a `# @default -- ` comment that explains the default schedule in English rather than just showing the somewhat opaque cron value. (Also probably worth mentioning in the description of the setting that this is a UNIX cron expression.)